Automattic/wp-calypso
# GF Signup: Dropdown Interval: Implement mobile sticky plan type selector behavior

Related to https://github.com/Automattic/growth-foundations/issues/296 and Attempt 1 at sticky headers https://github.com/Automattic/wp-calypso/pull/85158

## Proposed Changes

* Introduces the a sticky dropdown for mobile based screens

## Screenshots
![2023-12-21 06 14 59](https://github.com/Automattic/wp-calypso/assets/5414230/49d8eec2-47f2-46d5-925b-a80411099773)

## Testing Instructions



* Check out this branch
* Navigate to /start/plans?flags=onboarding/interval-dropdown ( don't forget the feature flag! )
* Verify that no changes are seen in tablet and desktop views
* Switch to a mobile view and verify that a sticky plan selector dropdown is displayed
* Verify that the dropdown updates mobile plan pricing as expected

## Pre-merge Checklist
